static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -399,7 +403,11 @@ define([
         mediator.on('page:media:moreinloaded', initPlayButtons);
 
         initFacia();
-        initMoreInSection();
+
+        if (config.page.section) {
```

Comment:
yeah I debated on putting it in there. My argument for putting it here was that the function name `initMoreInSection` sounds like it expects section to exist... it was a weak argument so happy to change.